test/CMakeLists.txt

```diff
@@ -61,3 +62,17 @@ else ()
 endif ()
 
 # FIXME: failing_with_issue is dead code :)
+
+check_cxx_compiler_flag("-fsanitize=fuzzer,address" HAS_FUZZ_FLAGS)
```

Comment:
What compiler(s) support this flag? We need documentation on this somewhere, at least in a comment here or elsewhere.

Reply:
As far as I'm aware if we were to break this up into two flags;
- `-fsanitize=fuzzer` supported by clang+msvc
- `-fsanitize=address` aka ASAN support by clang+gcc+msvc

Note that the actual flags for msvc are slightly different e.g. `\fsanitize` rather than `-fsanitize`

Reply:
I'll add a comment here :)

Reply:
> \fsanitize

...flags starting with a backslash, truly the end times are nigh :-)

Reply:
> > \fsanitize
> 
> ...flags starting with a backslash, truly the end times are nigh :-)

No, the MSVC flag starts with a `/` not a `\`.

Reply:
But AFAIK, MSVC understands `-` for the sake of compatibility, too.

Reply:
> What compiler(s) support this flag? We need documentation on this somewhere, at least in a comment here or elsewhere.

Part of the point of checking is that it's less necessary to know the complete matrix of compiler vendors and versions. We could add some `message(VERBOSE)`s for debugging when detection fails.

Reply:
Done in https://github.com/halide/Halide/pull/7512/files#diff-33394812ba204689144fd2f80832db83853ba1cb32403edb4e15fe4893e675fdR84-R86